tracpro/templates/frame.haml

```diff
@@ -39,6 +39,10 @@
     - block extra-less
     - block extra-style
 
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script>
```

Comment:
Should jquery ui need to be loaded on every page? I think we're only using it on a couple of them.